app/views/summary/aact.html.erb

```diff
@@ -5,6 +5,11 @@
     <h1>Studies Database Summary</h1>
   </div>
 
+  <div class="pb-3">
```

Comment:
Would be nice if you can add language "Current number of studies:" before the "ClinicalTrials.org:12412421 AACT:21215521"

Since we need to provide more readable information for the users.

Reply:
Thanks for the feedback!